test/dotcom/alerts/disruptions/subway_test.exs

```diff
@@ -79,20 +81,174 @@ defmodule Dotcom.Alerts.Disruptions.SubwayTest do
              } = future_disruptions()
     end
 
-    test "handles alert with more than one active_period" do
+    test "splits the active_periods of alerts with more than one into different buckets" do
       # Setup
+      {beginning_of_next_week, end_of_next_week} = service_range_next_week()
+
+      active_period_1_start =
+        Generators.DateTime.random_time_range_date_time(
+          {beginning_of_next_week, DateTime.shift(end_of_next_week, day: -2)}
+        )
+
+      active_period_1_end =
+        Generators.DateTime.random_time_range_date_time(
+          {active_period_1_start, DateTime.shift(end_of_next_week, day: -1)}
+        )
+
+      {beginning_of_week_after_next, _} = service_range_after_next_week()
+
+      active_period_2_start =
+        Generators.DateTime.random_time_range_date_time({beginning_of_week_after_next, nil})
+
+      active_period_2_end =
+        Generators.DateTime.random_time_range_date_time({active_period_2_start, nil})
+
```

Comment:
question: Why not have the second active period be just `service_range_after_next_week()`? Since you modified the first active period it won't get combined right?

Reply:
I admit that I'm new to Faker/communicative-tests/etc, but my understanding is that using `service_range_after_next_week()` would imply that there's something special about the active period starting right at the service range boundary, while this test doesn't actually care about that level of detail. 

(Also at least for right now, using `service_range_after_next_week()` would force me to deal with the indefinite-alert use case, which I didn't want to do yet 😇)

Reply:
It's fair, though we could also wonder whether there's something special about the active periods _not_ being at the boundary for these tests :P